docs/setup/install.md

```diff
@@ -67,36 +67,6 @@ To uninstall LeaderWorkerSet, run the following command:
 make undeploy
 ```
 
-# Install in a different namespace
-
-To install the leaderWorkerSet controller in a different namespace rather than `lws-system`, you should first:
-
-```sh
-git clone https://github.com/kubernetes-sigs/lws.git
-cd lws
-```
-
-Then change the [kustomization.yaml](../../config/default/kustomization.yaml) _namespace_ field as:
```

Comment:
If we want to install in another namespace, I guess this is still needed.

Reply:
https://github.com/kubernetes-sigs/lws/pull/285

Added that back. I did test by changing this before I opened this up for review.